### DISCUSSION TITLE
Calendar line break

### DISCUSSION THREAD

**Discussion by Enhed** [directus/directus#13506] [Feature Requests]
example: https://codepen.io/acerix/pen/eYpGJKq
Will be nice if `\n` works as linebreak in display template

![image](https://user-images.githubusercontent.com/36981278/170030462-02659eb4-567c-4e66-b672-41426b659dfe.png)
![image](https://user-images.githubusercontent.com/36981278/170030499-0cf4c6e7-1d07-4d65-a9f9-dfa1bf9995fc.png)

**Comment by Enhed** [comment]
@rijkvanzanten  what about that? =(

**Reply by rijkvanzanten** [reply]
I'm one of the 4 upvotes 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
